src/main/kotlin/me/clip/voteparty/cmds/CommandVoteParty.kt

```diff
@@ -29,16 +31,29 @@ internal class CommandVoteParty(override val plugin: VotePartyPlugin) : BaseComm
 {
 	
 	@Subcommand("addvote")
-	@Syntax("<amount>")
+	@Syntax("<amount> [player]")
 	@Description("Add Vote")
 	@CommandPermission(ADMIN_PERM)
-	fun addVote(issuer: CommandIssuer, @Default("1") amount: Int)
+	fun addVote(issuer: CommandIssuer, @Default("1") amount: Int, @Optional name: String?)
 	{
+		
 		if (amount <= 0)
 		{
 			return sendMessage(issuer, Messages.ERROR__INVALID_NUMBER)
 		}
 		
+		if (!name.isNullOrEmpty())
+		{
+			val user = party.usersHandler[name] ?: return sendMessage(issuer, Messages.ERROR__USER_NOT_FOUND)
+			
+			repeat(amount) {
+				val event = VoteReceivedEvent(user.player())
+				server.pluginManager.callEvent(event)
+			}
+			
+			sendMessage(issuer, Messages.VOTES__ADDED_TO_PLAYER, user.player(), "{count}", amount)
```

Comment:
This should probably don't also add votes directly after the event is called

Reply:
So what would you suggest as a modification? I get that calling the event will have it handle it, but if they don't put in a name, they just should auto-add the votes, right? Or should I throw that part in an "else" statement?

Reply:
Something like this https://paste.helpch.at/nazabizubu.bash

Reply:
`return sendMessage`

Reply:
Oh, awesome! I think it should be good now.